includes/Post_Types.php

```
@@ -125,4 +130,90 @@ public function register_taxonomy() {
+     *
+     * When a section or article is created under a vendor doc tree, it should
+     * automatically inherit the vendor doc flag from its root ancestor.
+     *
+     * @since WEDOCS_SINCE
+     *
+     * @param \WP_Post         $post    Inserted or updated post object.
+     * @param \WP_REST_Request $request Request object.
+     *
+     * @return void
+     */
+    public function inherit_vendor_doc_meta( $post, $request ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)